python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -283,7 +285,7 @@ def run(self, skyInfo, tempExpRefList, imageScalerList, weightList,
 
         Once the ``DcrModel`` reaches convergence or the maximum number of
         iterations has been reached, fill the metadata for each subfilter
-        image and make them proper ``coaddExposure``s.
+        image and make them proper ``coaddExposure`` .
```

Comment:
Not clear why you removed the plural 's' here?

Reply:
Anything with the format ``word``s causes the numpydoc build to complain.

Reply:
The solution is:

```
``coaddExposure``\ 's.
```

https://developer.lsst.io/restructuredtext/style.html#inline-text-styling

Reply:
...and then you need to tell python it's a `r"""` string to prevent warnings about unknown escapes.